.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+    - main
+    paths:
+      - 'google-cloud-bigtable/**'
+  pull_request:
+    paths:
+      - 'google-cloud-bigtable/**'
+name: conformance
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby_version: [
+          2.7,
+          3.0,
```

Comment:
Surround each of the version numbers with quotes to make them strings instead of numbers. Fixes an issue where json can parse a ".0" version as an integer and drop the minor version.